administration/intro.adoc

```diff
@@ -180,7 +181,13 @@ The following
 https://docs.openshift.com/container-platform/3.7/admin_guide/manage_scc.html[SCC]
 needs to be added prior KubeVirt deployment:
 
-- When you use the stable deployment flow:
+- When you use the operator flow:
+[source,bash]
+----
+$ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-operator
+----
+
+- When you use the alternative deployment flow
 ____________________________________
 Important: Starting with version v0.15.0 please replace `kubevirt-privileged` with `kubevirt-handler`
```

Comment:
we should probably have it documented vice versa: have kubevirt-handler in the CLI example above and (maybe, not sure) document kubevirt-privileged workaround for older versions. (Though again I would prefer if we stop documenting old versions.)

Reply:
done